genesyscloud/provider/provider.go

```diff
@@ -268,14 +268,23 @@ func InitClientConfig(data *schema.ResourceData, version string, config *platfor
 		RetryWaitMax: time.Second * 30,
 		RetryMax:     20,
 		RequestLogHook: func(request *http.Request, count int) {
-			if count > 0 && request != nil {
-				log.Printf("Retry #%d for %s %s", count, request.Method, request.URL)
+			sdkDebugRequest := newSDKDebugRequest(request, count)
+			request.Header.Set("TF-Correlation-Id", sdkDebugRequest.TransactionId)
```

Comment:
Not sure why it picked this up.  Most have branched off a branch.  This was addressed in another PR.